### PR TITLE
feat(http): Allow Passing the exact same path and domain attributes while removing cookies

### PR DESCRIPTION
**Pull request by getspooky**

### CODE REVIEW

File: http/cookie_test.ts [hunk 1]
```diff
@@ -3,6 +3,13 @@ import { Response, ServerRequest } from "./server.ts";
 import { deleteCookie, getCookies, setCookie } from "./cookie.ts";
 import { assert, assertEquals, assertThrows } from "../testing/asserts.ts";
 
+Deno.test({
+  name: "Cookie response",
+  fn(): void {
+    //
+  },
+});
+
```
Comment (bartlomieju) [inline http/cookie_test.ts:12]: Seems like left by mistake?

**Review by bartlomieju — COMMENTED** [review]
Thanks for the PR @getspooky, does it fix some concrete problem?

File: http/cookie_test.ts (hunk 1 above)
Comment (getspooky) [reply http/cookie_test.ts]: yes, fixed

**Comment by getspooky** [comment]
@bartlomieju  let's say
 ```ts
setCookie({ headers }, { name: "deno", value: "cat"});
setCookie({ headers }, { name: "deno", value: "cat", path: "/demo" });
```
if we run 

```ts
deleteCookie({ headers }, "deno");
```
it will not remove both cookies , so the puporse of this PR is to provide a way to remove cookie by specify path or domain.

for example 

```ts
deleteCookie({ headers }, "deno", { path: "/demo" });
```

**Review by bartlomieju — APPROVED** [review]

**Pull request merged**